### PR TITLE
Mocked eth1 data for interop

### PR DESCRIPTION
**Pull request by mpetrunic**
I've commented out tests for real eth1 data as it's time consuming to rewrite them. We can fix those stuff after interop.

### CODE REVIEW

**Review by GregTheGreek — DISMISSED** [review]

**Review by wemeetagain — DISMISSED** [review]
looks good! :+1:

**Review by wemeetagain — APPROVED** [review]

File: packages/lodestar/src/eth1/impl/ethers.ts [hunk 1]
```diff
@@ -138,6 +139,41 @@ export class EthersEth1Notifier extends (EventEmitter as { new(): Eth1EventEmitt
     return Buffer.from(depositCountHex.substr(2), 'hex').readUIntLE(0, 6);
   }
 
+  public async getEth1Data(config: IBeaconConfig, state: BeaconState, currentEpoch: Epoch): Promise<Eth1Data> {
+    const [head, latestStateBlock] = await Promise.all([
+      this.getHead(),
+      this.getBlock('0x' + state.eth1Data.blockHash.toString('hex'))
+    ]);
+    const validVotes = await this.filterValidVotes(config, state.eth1DataVotes, head, latestStateBlock);
```
Comment (GregTheGreek) [inline packages/lodestar/src/eth1/impl/ethers.ts:147]: can we get a type?

File: packages/lodestar/src/eth1/impl/ethers.ts [hunk 2]
```diff
@@ -190,4 +226,29 @@ export class EthersEth1Notifier extends (EventEmitter as { new(): Eth1EventEmitt
       },
     };
   }
+
+  private async filterValidVotes(
+    config: IBeaconConfig,
+    votes: Eth1Data[],
+    head: Block,
+    latestStateBlock: Block): Promise<Eth1Data[]> {
+    const potentialVotes = [];
```
Comment (GregTheGreek) [inline packages/lodestar/src/eth1/impl/ethers.ts:235]: here as well

**Review by GregTheGreek — CHANGES_REQUESTED** [review]
Aside from me being picky, LGTM :)

File: packages/lodestar/src/eth1/impl/ethers.ts (hunk 1 above)
Comment (mpetrunic) [reply packages/lodestar/src/eth1/impl/ethers.ts:147]: it will get type automatically though

Comment (GregTheGreek) [reply packages/lodestar/src/eth1/impl/ethers.ts:147]: hmm... do we have an interface defined for `vote`?

Comment (mpetrunic) [reply packages/lodestar/src/eth1/impl/ethers.ts:147]: vote is Eth1Data object.

filterValidVotes methods returns promise of Eth1Data array

Comment (GregTheGreek) [reply packages/lodestar/src/eth1/impl/ethers.ts:147]: ah ok

**Review by GregTheGreek — APPROVED** [review]

**Pull request merged**